frontend/public/module/k8s/k8s-reducers.ts

```diff
@@ -35,9 +38,7 @@ const updateList = (list, nextJS) => {
 
   // TODO: (kans) only store the data for things we display ...
   //  and then only do this comparison for the same stuff!
-  const currentJS = current.toJSON();
-  currentJS.metadata.resourceVersion = nextJS.metadata.resourceVersion;
-  if (_.isEqual(currentJS, nextJS)) {
+  if (current.deleteIn(['metadata', 'resourceVersion']).equals(next.deleteIn(['metadata', 'resourceVersion']))) {
```

Comment:
I tried this once a while ago, and it didn't work.  I must have used `Map` for my test instead of`fromJS` - the former only works if the nested objects are `===`.